src/test_model_parser.cc

```diff
@@ -180,8 +180,7 @@ TEST_CASE("ModelParser: test Init* functions")
 
   SUBCASE("InitOpenAI")
   {
-    status = mmp.InitOpenAI(
-        model_name, model_version, batch_size, InferenceLoadMode::Concurrency);
+    status = mmp.InitOpenAI(model_name, model_version, batch_size);
```

Comment:
As mentioned [here](https://github.com/triton-inference-server/perf_analyzer/pull/366/files#r2058832731), `ModelParser::InitOpenAI` no longer takes the inference load mode, instead inference-load-mode-related input logic is handled in the `ModelParser` constructor.

But regardless, for some reason, this test didn't actually do anything differently based on the inference load mode.